### PR TITLE
🔧 migrate b-navbar

### LINKED ISSUE #6395: Navbar modal won't close when clicking explore items or collections
### What happened?

obs: Same happens for other menus like "Create"

https://github.com/kodadot/nft-gallery/assets/36627808/85315ba8-15ca-4ec3-95bb-4939bfa0184a

### Please reproduce in steps

Hop on mobile, connected or unconnected, go to menu -> explore, wait for the page to load, go to menu again, click explore or items

### Expected Behavior

Modal should close after you click one of the options

### What browsers are you seeing the problem on?

Desktop Chrome, Desktop Firefox

### At which address did you encounter bug?

canary.kodadot.xyz, beta.kodadot.xyz

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

^

### Relevant log output

_No response_

### Payment link for reward

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by roiLeo**
- part of #5852 
- closes #6395 

@prury you have to check if nothing breaks in navbar (mobile, desktop, links etc..) compared to beta/canary

<s>edit: Mobile navbar doesn't work hehe</s> fixed 🎉 

note: Didn't changed `StatsDropdown` component since it's not used

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -1,149 +1,180 @@
...
-    <template #brand>
-      <b-navbar-item :to="{ path: '/' }" class="logo" tag="nuxt-link">
-        <img
-          :src="logoSrc"
-          alt="First NFT market explorer on Kusama and Polkadot"
-          width="143"
-          height="42" />
-      </b-navbar-item>
+  <nav
+    drole="navigation"
```
Comment (daiagi) [inline components/Navbar.vue:3]:
```suggestion
role="navigation"
```

**Review by daiagi — APPROVED** [review]
looks good
didn't find anything that isn't working

**Comment by prury** [comment]
**Mobile**:

Javascript chain 😄 


---------
When you are disconnected on mobile and try to connect, the burger menu is not closing to allow the user to connect:

https://github.com/kodadot/nft-gallery/assets/36627808/cabc3635-27ab-4739-a5ad-72e52c40bb1b

---------
Language button when disconnected on mobile: 
it returns a 404 page or it refreshes and don't change the language

when those get fixed I'll do another round of testing

**Comment by roiLeo** [comment]
Blocked: 

`TypeError: can't access property "abstract", parent.$options is undefined` 
related with #6277

**Comment by prury** [comment]
@roiLeo switching language on mobile changes the language but the modal won't close as soon as i click it, it keeps open.

**Comment by roiLeo** [comment]
> @roiLeo switching language on mobile changes the language but the modal won't close as soon as i click it, it keeps open.

Isn't it the current behavior on beta? are you logged in?

**Comment by prury** [comment]
> > @roiLeo switching language on mobile changes the language but the modal won't close as soon as i click it, it keeps open.
> 
> Isn't it the current behavior on beta? are you logged in?

i'm not logged and using mobile

beta behavior:


https://github.com/kodadot/nft-gallery/assets/36627808/de7188e0-3edd-48ec-96ac-dd54df9e1c5d

deploy:


https://github.com/kodadot/nft-gallery/assets/36627808/1a74ba98-c6b6-49c1-a79c-27b240becff9

**Comment by roiLeo** [comment]
Should be good now, thanks for reporting

**Comment by prury** [comment]
> Should be good now, thanks for reporting

ty, will do another round of testing now

**Comment by prury** [comment]
- scroll / search bar button issue(only happens on main page): 

https://github.com/kodadot/nft-gallery/assets/36627808/0b803d39-810d-402f-afe8-5998c0d9a886

- Search button opening the entire modal (beta only shows the search bar itself): 

https://github.com/kodadot/nft-gallery/assets/36627808/f1d4340b-6894-4355-a075-8ed9119d5e57

found a old bug: #6395 

for now i think that's it, but i wouldn't mind testing a bit more later

**Comment by roiLeo** [comment]
closes #6395, feel free to test again

**Comment by prury** [comment]
thanks a lot for the fixes @roiLeo 

only problem i see now is this one: 
https://github.com/kodadot/nft-gallery/assets/36627808/faba6ece-ddc4-4f8b-9dab-c18c6a7610d8

search icon not appearing on other pages, only inside menu

**Pull request merged**